src/core/tests/conditional_compilation/ngraph_cc_on.cpp

```diff
@@ -23,18 +26,30 @@ using namespace std;
 TEST(conditional_compilation, disabled_op_scope) {
 #define ov_op_Scope0 1
     int n = 0;
-    const std::string errMsg = "ngraph_op_Scope1 is disabled!";
 
-    // Simple scope is enabled
+    // Simple Scope0 is enabled
     OV_OP_SCOPE(Scope0);
     n = 42;
     EXPECT_EQ(n, 42);
 
-    // Simple scope is disabled
+    // Simple Scope1 is disabled and throws exception
     ASSERT_THROW(OV_OP_SCOPE(Scope1), ngraph::ngraph_error);
 #undef ov_op_Scope0
 }
 
+TEST(conditional_compilation, disabled_Constant_in_opset) {
+    ov::OpSet opset("test_opset3");
+#define ov_opset_test_opset3_Abs 1
+    INSERT_OP(test_opset3, Abs, ov::op::v0);
+    EXPECT_NE(opset.create("Abs"), nullptr);
+    EXPECT_NE(opset.create_insensitive("Abs"), nullptr);
+#undef ov_opset_test_opset3_Abs
+
+    INSERT_OP(test_opset3, Constant, ov::op::v0);
+    EXPECT_EQ(opset.create("Constant"), nullptr);
+    EXPECT_EQ(opset.create_insensitive("Constant"), nullptr);
```

Comment:
```suggestion
#define ov_opset_test_opset3_Abs 1
    ov::OpSet opset("test_opset3");
    INSERT_OP(test_opset3, Abs, ov::op::v0);
    EXPECT_NE(opset.create("Abs"), nullptr);
    EXPECT_NE(opset.create_insensitive("Abs"), nullptr);

    INSERT_OP(test_opset3, Constant, ov::op::v0);
    EXPECT_EQ(opset.create("Constant"), nullptr);
    EXPECT_EQ(opset.create_insensitive("Constant"), nullptr);
#undef ov_opset_test_opset3_Abs
```

Reply:
The idea was to split behaviour per Op for more visibility, so I'd prefer def/undef macros only per Op, not at start/finish of the test